9.4/root/usr/bin/run-postgresql

```diff
@@ -2,8 +2,9 @@
 
 set -eu
 
-source ${HOME}/common.sh
+source "$CONTAINER_SCRIPTS_PATH"/common.sh
```

Comment:
https://google.github.io/styleguide/shell.xml?showone=Quoting#Quoting

I'm usually follow this :-) Maybe it is wrong, but they say:

```
# "quote variables"
echo "${flag}"
```